src/validationinterface.cpp

```diff
@@ -198,7 +197,7 @@ void CMainSignals::UpdatedBlockTip(const CBlockIndex *pindexNew, const CBlockInd
     auto event = [pindexNew, pindexFork, fInitialDownload, this] {
         m_internals->Iterate([&](CValidationInterface& callbacks) { callbacks.UpdatedBlockTip(pindexNew, pindexFork, fInitialDownload); });
     };
-    ENQUEUE_AND_LOG_EVENT(event, "%s: new block hash=%s fork block hash=%s (in IBD=%s)", __func__,
+    ENQUEUE_AND_LOG_EVENT(BCLog::VALIDATION, BCLog::Level::Info, event, "new block hash=%s fork block hash=%s (in IBD=%s)",
```

Comment:
Also seems pointless to upgrade these to `Info`... If anything, these seem more appropriate to downgrade to `Trace` logging (the "Enqueuing" particularly).

Reply:
I'll switch these to `Debug` and downgrade the `Enqueuing` wrapper to trace.